conf/application.conf

```diff
@@ -95,6 +95,14 @@ api {
     auth-validation.enabled = true
   }
 
+  supporting-agent-endpoints {
+    create-amend-foreign=false
+    delete-foreign=false
+    retrieve-foreign=false
+
```

Comment:
Could this be tidied a bit? Remove the blank line, & add spaces so "blah-blah = false"